wallet/wallet.go

```diff
@@ -4095,3 +3972,186 @@ func OpenWithRetry(db walletdb.DB, pubPass []byte, cbs *waddrmgr.OpenCallbacks,
 
 	return w, nil
 }
+
+// MapBroadcastBackendError maps the different backend errors when broadcasting
+// a transaction to the bitcoin network to an internal error type.
+func MapBroadcastBackendError(err error) error {
```

Comment:
Is it intended to export this method?

Reply:
yes because we wanna reuse the mapping code for the Rebroadcaster which is currently in the neutrino package.

Reply:
Yep, goal here is to be able to have the rebroadcaster gain a custom hook to handle errors other than p2p errors.